tools/rfove/rfove.xml

```diff
@@ -0,0 +1,51 @@
+<tool id="rfove" name="Perform segmentation region-based fitting of overlapping ellipses" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.05">
+    <description>with RFOVE</description>
+    <macros>
+        <token name="@TOOL_VERSION@">2023.11.12</token>
+        <token name="@VERSION_SUFFIX@">1</token>
+    </macros>
+    <edam_operations>
+        <edam_operation>operation_3443</edam_operation>
+    </edam_operations>
+    <xrefs>
+        <xref type="bio.tools">rfove</xref>
+    </xrefs>
+    <requirements> 
+        <container type="docker">docker.io/kostrykin/rfove:@TOOL_VERSION@</container>
+    </requirements>
+    <command detect_errors="aggressive">
+    <![CDATA[
+    #set $neighborhood_size = "%d" % ($half_neighborhood_size.value * 2 + 1)
+    echo `uname -a` >> /tmp/hostinfo.txt &&
+    /rfove
+        ${area_ub}
+        ${min_area_max_area_ratio_ub}
+        ${overlap_ub}
+        ${neighborhood_size}
+        ${dataset}
+        output.tiff
+    ]]>
+    </command>
+    <inputs>
+        <param name="dataset" type="data" format="tiff,png" label="Dataset" />
+        <param name="area_ub" type="integer" label="Maximum ellipse area" value="250" min="5" />
+        <param name="min_area_max_area_ratio_ub" type="float" label="Upper bound of the ratio: minimum area / maximum area" value="0.1" min="0" max="1" />

```

Comment:
https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#coding-style

```suggestion
        <param name="min_area_max_area_ratio_ub" type="float" min="0" max="1" value="0.1" label="Upper bound of the ratio: minimum area / maximum area" />

```

We have some recommendation for the ordering.